src/core/MythosArea.ttslua

```diff
@@ -35,13 +36,37 @@ end
 function onLoad(savedData)
   if savedData and savedData ~= "" then
     local loadedState = JSON.decode(savedData) or {}
+    scenarioCard      = getObjectFromGUID(loadedState.scenarioCardGUID) or {}
     currentScenario   = loadedState.currentScenario or ""
     useFrontData      = loadedState.useFrontData or true
     tokenData         = loadedState.tokenData or {}
   end
   Wait.time(function() collisionEnabled = true end, 0.1)
 end
 
+function copyScenarioReferenceCard()
+  if scenarioCard ~= nil then
+    usedColors = playermatApi.getUsedMatColors()
+    if #usedColors ~= 0 then
+      for _, color in ipairs(usedColors) do
+        if color == "Green" then
+          scenarioCard.clone({position = {x = -39.12, y = 1.55, z = 22.58}, snap_to_grid = true})
+        elseif color == "Red" then
+          scenarioCard.clone({position = {x = -21.57, y = 1.55, z = -22.58}, snap_to_grid = true})
+        elseif color == "Orange" then
+          scenarioCard.clone({position = {x = -50.98, y = 1.55, z = -24.87}, snap_to_grid = true})
+        else
+          scenarioCard.clone({position = {x = -50.98, y = 1.55, z = 7.32}, snap_to_grid = true})
+        end
+      end
+    else
+      broadcastToAll("No investigators placed.", "Red")
+    end
+  else
+    broadcastToAll("No scenario reference card found.", "Red")
```

Comment:
Code would be more readable if this was at the top (and then that section ended with a `return` - that way you don't have this nesting)